pico/README.md

```diff
@@ -30,18 +30,28 @@ sudo git -C /opt/pico-sdk submodule update --init
 export PICO_SDK_PATH=/opt/pico-sdk
 ```
 
-## Build the firmware
+## Build the firmware for Apple IIe
 ```shell
 cd /AppleII-VGA/pico
 mkdir build
 cd build
-cmake -DCMAKE_BUILD_TYPE=Release ..
+cmake -DAPPLE_MODEL=IIE -DCMAKE_BUILD_TYPE=Release ..
+make
+$ ls *uf2
+...
+applevga.uf2
+```
+## Build the firmware for Apple II+
+```shell
+cd /AppleII-VGA/pico
+mkdir build
+cd build
+cmake -DAPPLE_MODEL=II+ -DCMAKE_BUILD_TYPE=Release ..

```

Comment:
Thanks for adding to the README!  This should be `-DAPPLE_MODEL=IIPLUS`

Reply:
Done